### PR TITLE
Example Parachain Integration tests

### PR DESCRIPTION
**Pull request by AlexD10S**
Solved the issue [QA] Example Parachain Integration tests https://github.com/paritytech/extended-parachain-template/issues/77.

Added the instructions of how to run the integration tests, set up the zombienet file and two tests files: 

- `force_hrmp_open_channels`: creates an HRMP channel with asset-hub and checks that everything is created propertly.

I decided to create this test, because there currently there is an issue: (draft) Zombienet test with preopen hrmp with zombienet investigation https://github.com/paritytech/polkadot-sdk/pull/1616 that is unable to create the hrmp channels directly with zombienet, so this test can be useful.

- `transact`: Test to make a transfer from this parachain sovereign account to asset-hub local network sovereign account via a XCM Transact in the Relay Chain.

Bonus point:
We can create one reserve transfer similar [to the one in Trappist ](https://github.com/paritytech/trappist/tree/main/integration-tests) once we have this: Minimal AssetHub integration https://github.com/paritytech/extended-parachain-template/issues/79

### CODE REVIEW

File: integration-tests/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+```
+$ yarn global add ts-node
+
+$ yarn global add @parity/parachains-integration-tests
+```
+
+Review the [Run a local network](https://github.com/paritytech/extended-parachain-template#%EF%B8%8F-run-a-local-network) to set up Zombienet in your system.
+
+Create a `bin` directory into the root of this repository and place the following binaries inside of it:
+- `polkadot` (which you can download from [the releases](https://github.com/paritytech/polkadot/releases))
```
Comment (weezy20) [inline integration-tests/README.md:17]: For polkadot-sdk (After #85) we would additionally need `polkadot-prepare-worker` and `polkadot-start-worker` binaries inside `bin/`

Comment (AlexD10S) [reply integration-tests/README.md:17]: True, I have them. I will document it

File: zombienet-config/integration-tests-config.toml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  validator = true
+
+[[parachains]]
+id = 1000
+add_to_genesis = true
+cumulus_based = true
+chain = "statemine-local"
+
+  [[parachains.collators]]
+  name = "statemine-collator01"
```
Comment (weezy20) [inline zombienet-config/integration-tests-config.toml:36]: name = "asset-hub-collator01"

File: integration-tests/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- `polkadot-parachain` (which you will build from [cumulus](https://github.com/paritytech/cumulus))
```
Comment (weezy20) [inline integration-tests/README.md:19]: Why not use the EPT binary here?

File: zombienet-config/integration-tests-config.toml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+  [[relaychain.nodes]]
+  name = "dave"
+  validator = true
+
+[[parachains]]
+id = 1000
+add_to_genesis = true
+cumulus_based = true
+chain = "statemine-local"
```
[the remark this answers is not in the document]
Comment (AlexD10S) [reply zombienet-config/integration-tests-config.toml]: Done! to `asset-hub-kusama-local` that is the equivalent to `statemine-local`

File: zombienet-config/integration-tests-config.toml (hunk 2 above)
Comment (AlexD10S) [reply zombienet-config/integration-tests-config.toml]: Done!

File: integration-tests/force_hrmp_open_channels.yml [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+    local_parachain:
+      wsPort: 9920
+      paraId: &lp_id 2000
+  variables:
+    common:
+      amount: &amount 20000000000000
+      hrmp_channels:
+        proposed_max_capacity: &max_capacity 8
+        proposed_max_message_size: &max_message_size 1048576
+        channel: &channel {
```
Comment (weezy20) [inline integration-tests/force_hrmp_open_channels.yml:18]: I don't think we are using this variable binding anywhere. Okay to omit it?

**Comment by weezy20** [comment]
Looks good to me.

**Review by weezy20 — APPROVED** [review]

**Pull request merged**